python/tests/test_highlevel.py

```diff
@@ -1234,6 +1234,14 @@ def test_get_time(self):
             for u in range(N):
                 self.assertEqual(ts.get_time(u), ts.node(u).time)
 
+    def test_oldest_time(self):
+        for ts in get_example_tree_sequences():
+            oldest = 0
+            for tree in ts.trees():
+                for root in tree.roots:
+                    oldest = max(oldest, tree.time(root))
+            assert oldest == ts.oldest_time
```

Comment:
Must use unittest methods as we're using node, e.g., ``self.assertEqual(oldest, ts.oldest_time)``